app/views/groups/_info.html.erb

```diff
@@ -33,7 +33,8 @@
           dataConfirm: t('common.actions.confirm'),
           dataMethod: 'delete'
         } : nil
-      }
+      },
+      storyName: local_assigns[:group].description
```

Comment:
I think we want to use the Group name here instead of the description, which could be long.

```suggestion
      storyName: local_assigns[:group].name
```